docs/reference/credentials.md

```diff
@@ -0,0 +1,104 @@
+# Credentials
+
+AnyCloud deployments expect the cloud credentials to be configured using a local file that is not committed to any repository and is located at `~/.anycloud/credentials.json`. Each deployment will be keyed by a name and will provide the cloud provider configuration via two fields:  `credentials` and `cloudProvider`. Possible values for `cloudProvider` are `AWS`, `GCP` and `Azure`. `credentials` will have a different schema depending on which cloud provider it is.
+
+## AWS
+
+The schema for the AWS deployment config is as follows:
+
+```javascript
+{
+  "string": {
+    "cloudProvider": "string",
+    "credentials": {
+      "accessKeyId": "string",
+      "secretAccessKey": "string"
+    }
+  },
+  ...
+}
+```
+
+The `accessKeyId` and `secretAccessKey` come from an IAM user with an [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess)policy attached. An example credential entry would look like this:
+
+```javascript
+{
+  "piedpiper-aws": {
+    "cloudProvider": "AWS",
+    "credentials": {
+      "accessKeyId": "#####################",
+      "secretAccessKey": "###################"
+    },
+  },
+  ...
+}
+```
```

Comment:
I think you can condense this quite a bit and get everything across:

```json
{
  "piedpiper-aws": {
    "cloudProvider": "AWS",
    "credentials": {
      "accessKeyId": "#############",
      "secretAccessKey": "#############"
    }
}
```

The top-level key is the alias you provide for referring to these credentials. For AWS the `cloudProvider` value is `AWS`, and in the credentials the `accessKeyId` and `secretAccessKey` come from an IAM user with an [`AmazonEC2FullAccess`](https://console.aws.amazon.com/iam/home#/policies/arn%3Aaws%3Aiam%3A%3Aaws%3Apolicy%2FAmazonEC2FullAccess)policy attached.


This takes up less space while explaining more at the same time, and the other examples below can also be reworked similarly.